tests/test-upgrade.py

```diff
@@ -24,6 +24,13 @@ def test_upgrade(self):
         cmd = "sudo snap install microk8s --classic --channel={}".format(upgrade_from).split()
         check_call(cmd)
         wait_for_installation()
+        if is_container():
+            # In some setups (eg LXC on GCE) the hashsize nf_conntrack file under
+            # sys is marked as rw but any update on it is failing causing kube-proxy
+            # to fail.
```

Comment:
So we call path-kube-proxy, which will tell kube-proxy to not use conntrack.

Reply:
Easy